TelCo.ColorCoder/ColorCodeManual.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    internal class ColorCodeManual : ColorPair
+    {
+        internal int pairNo;
+
+        /// <summary>
+        /// function returns all the ColorCode pairs mapped with pairNo
+        /// </summary>
+        /// <param name="colorMapMajor">Array of Major color</param>
+        /// <param name="colorMapMinor">Array of Minor color</param>
+        /// <returns></returns>
+        public static List<ColorCodeManual> GetColorCodeManual(Color[] colorMapMajor, Color[] colorMapMinor)
```

Comment:
Good separation of new functionality into its own file

Reply:
Thank you sir.